lib/process-services-cloud/src/lib/task/task-header/components/task-header-cloud.component.ts

```diff
@@ -112,7 +114,7 @@ export class TaskHeaderCloudComponent implements OnInit, OnDestroy, OnChanges {
                     this.refreshData();
                 }
             },
-            (err) => this.taskError.emit(err), () => {});
+            (err) => this.error.emit(err), () => {});
```

Comment:
you don't need the last empty block
```suggestion
            (err) => this.error.emit(err));
```

Reply:
done :)